### PR TITLE
Integrate Chakra UI Kit

### LINKED ISSUE #3: Integrate Chakra UI Kit

### PR DESCRIPTION
**Pull request by ctc-devops**
closes #3

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <ChakraProvider>
```
Comment (meghakak) [inline src/App.jsx:8]: Super cool

**Review by meghakak — CHANGES_REQUESTED** [review]
See comments

File: src/App.jsx (hunk 1 above)
Comment (meghakak) [reply src/App.jsx:8]: Resolving

Comment (sylvesteryue) [reply src/App.jsx:8]: sheeesh

**Review by meghakak — APPROVED** [review]
LGTM

**Review by sylvesteryue — APPROVED** [review]
LGTM

**Pull request merged**